stage1/tasks/hangman/README.md

```diff
@@ -0,0 +1,88 @@
+*This example is purely illustrative. Feel free to change the layout as you wish.*
+![screenshot](hangman.png)
+
+### Helpful materials
+At the [following link](https://www.figma.com/file/ug2NAUiXPpaFDvch5TWUxd/Hangman-game?type=design&node-id=0%3A1&mode=design&t=4Gj7Ayo0yckppNo8-1) you will find:
+- examples of the game flow,
+- body parts (for those who don't want to design them by themselves).
+
+### Main functional requirements
+- The design should be adaptive (or responsive), that includes desktop(1440px <= width), tablet(768px <= width < 1440px) and mobile(360px <= width < 768px). When switching between versions everything should be displayed correctly, all functionality should be present, nothing should disappear or leave beyond the screen.
+- Initially, body in the index.html file must be empty, all necessary elements are generated using JS.
+- At the beginning of the game, the question-answer pair should be chosen randomly.
+- The application can be done in English or in Russian language.
+
+## Repository requirements
+- The task should be done in private school's repository.
+- The code should be committed to a separate branch named as the task.
+- The `main` (or old style master) branch should be empty (contain only files like README.md or .gitignore).
+- Commit messages should follow the [guideline](https://docs.rs.school/#/git-convention)
+- Once the work is finished, create a pull request from a **development** branch to **main** (or master).
+- The pull request name should contain **the task name**.
+- The pull request description should be provided according to the [template](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr).
+
+## Technical requirements
+- The application is displayed correctly and functions properly in the latest version of Google Chrome.
+- It is **not allowed** to use:
+    - CSS frameworks (e.g., `Bootstrap`),
+    - JS frameworks (e.g., `Angular`, `React`, `Vue`, etc.),
+    - Outdated libraries (e.g. `JQuery`, etc.),
+    - `TypeScript`.
+- You can use CSS preprocessors (`SASS`, `SCSS`) and `normalize.css`.
+- Using JS frameworks (e.g., Angular, React, Vue, etc.) is not allowed.
+
+## CrossCheck Criteria (150 points)
+*It is recommended to print the secret word in the browser's console to facilitate the cross-check process.*
+- [ ] Responsive/adaptive UI from 1440px to 360px viewport: `+10`
+- [ ] The generation of DOM elements is implemented. `body` in the `index.html` is empty (can contain only `script` tag): `+20`
+- [ ] The game starts with the correct default view (empty gallows, blank lines for secret word, etc.) and a random question: `+5`
+- [ ] The user can play the game by using the virtual keyboard: `+20`
+- [ ] The user can play the game by using the physical keyboard: `+20`
+- [ ] When the letter is correct, it appears instead of the corresponding blank line. If the letter repeats in the word, all corresponding blank lines should be replaced by it: `+15`
```

Comment:
_The same suggestion about the word "underscores" and "must"_

**Suggested:** - [ ] When the letter is correct, it appears instead of the corresponding underscore. If the letter repeats in the word, all corresponding underscores must be replaced by it: `+15`

Reply:
Done